src/main/java/core/basesyntax/service/ReportService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public interface ReportService {
```

Comment:
Consider adding Javadoc comments to the interface to describe its purpose and functionality.